src/viewer/viewer.ts

```diff
@@ -1,6 +1,5 @@
-import path from 'path';
 import * as vscode from 'vscode';
-import * as utils from './utils';
+import * as utils from '../utils';
 
 export class ViewerCustomEditor implements vscode.CustomReadonlyEditorProvider {
 	public static readonly viewType = 'gryc.editor';
```

Comment:
Similarly, we should just keep `gryc.gallery` and `gryc.viewer`. I think `gryc.editor` is redundant.

But TBH I'm not sure what's the role of `viewType`...

Reply:
Yea I agree. `viewType` is for activation events to identify which view is needs to be displayed